README.md

```diff
@@ -30,10 +30,10 @@ The quickest way to add VexFlow to a web page is via a `<script>` tag.
 <script src="https://cdn.jsdelivr.net/npm/vxflw-early-access@alpha/build/cjs/vexflow.js"></script>
 
 <script>
-document.fonts.ready.then(() => {
-  Vex.Flow.setMusicFont('Bravura', 'Academico');
-  // YOUR CODE GOES HERE
-});
+  document.fonts.ready.then(() => {
+    Vex.Flow.setFont('Bravura', 'Academico');
```

Comment:
Thanks. That's good feedback.

Which of the following is best?

```
setFonts(...)
setFontFamily(...)   // since the underlying code modifies MetricsDefaults.fontFamily
setFontStack(...)
```

I am OK with setFonts / getFonts if you are OK with it, since it is short. But maybe `setFontStack()` is the clearest naming?

Reply:
We should look into this in a separate PR:

This method modifies `MetricsDefaults.fontFamily`. I assumed `MetricsDefaults` would just be the initial defaults, and that the Metrics class would have a setter method that would modify its internal dictionary. How come calling setFonts() modifies the defaults object?

Reply:
Probably MetricsDefault is not the right name. I would go for get/setFonts as we already have loadFonts and it is also shorter.

Reply:
Done. Renamed to getFonts()/setFonts().